testing/README.md

```diff
@@ -0,0 +1,34 @@
+# Spanner-Hibernate Integration Testing
+
+This directory contains a script which allows you to run the Spanner-Hibernate dialect against
+the integration tests provided in the [Hibernate ORM repository](https://github.com/hibernate/hibernate-orm).
+
+The script will take your local working copy of `google-cloud-spanner-hibernate` and install to
+local maven in order to facilitate testing, so it will allow you to test local changes.
+
+## Instructions
+
+1. Install Python3 on your machine.
+
+    Verify you have it installed by running:
+    
+    ```SHELL
+    $ python3 --version
```

Comment:
Seeing the final script, do you still feel writing it in Python has an advantage?

Reply:
Yeah, I see the possibility we expand on this in the future, so I think this is going to be easier to maintain.

Plus in the current script, there is some minor complexity with checking the existing subdirectories to omit cloning the repo if it already exists. I think it is already worth using Python to avoid writing that in bash.